lib/matplotlib/tests/test_simplification.py

```diff
@@ -13,11 +15,13 @@
 from matplotlib.path import Path
 
 
-# NOTE: All of these tests assume that path.simplify is set to True
-# (the default)
+# NOTE: All of these tests used to assume that path.simplify is set to True
```

Comment:
Perhaps just remove the comment?
You may also be able to just use a module-level autouse fixture to set the rcparam (see https://docs.pytest.org/en/latest/fixture.html).